transport/src/test/java/io/camunda/zeebe/transport/stream/impl/RemoteStreamPusherTest.java

```diff
@@ -29,7 +29,7 @@ final class RemoteStreamPusherTest {
   private final TestTransport transport = new TestTransport();
   private final Executor executor = Runnable::run;
   private final RemoteStreamPusher<Payload> pusher =
-      new RemoteStreamPusher<>(streamId, transport, executor);
+      new RemoteStreamPusher<>(streamId, transport, executor, new RemoteStreamMetrics() {});
```

Comment:
:wrench: As this occurs in a bunch of places, we could make a Noop implementation available.